project-base/infrastructure/google-cloud/stateful-services.tf

```diff
@@ -1,6 +1,7 @@
 resource "google_storage_bucket" "file-store" {
   name     = "${var.GOOGLE_CLOUD_STORAGE_BUCKET_NAME}"
   location = "EU"
+  force_destroy = true
```

Comment:
could we put in documentation that if someone would like to keep the data, it needs to be backed up ?